python/hail/docs/tutorials/03-expressions.ipynb

```diff
@@ -76,9 +167,9 @@
     }
    },
    "source": [
-    "In order to process datasets of this size, Hail uses lazy evaluation.\n",
+    "Eager evaluation won't work on datasets that won't fit in memory. Consider the UK Biobank BGEN file, which is ~2TB but decompresses to >100TB in memory.\n",
     "\n",
-    "When you enter an expression, Hail doesn't execute the expression immediately: it simply records what you asked to do."
+    "In order to process datasets of this size, Hail uses **lazy** evaluation. When you enter an expression, Hail doesn't execute the expression immediately: it simply records what you asked to do."
```

Comment:
I think a semi-colon or two dashes is correct over a colon separating the two phrases.